Assets/MixedRealityToolkit.Extensions/Webrtc/Scripts/Marshalling/NativePeer.cs

```diff
@@ -0,0 +1,470 @@
+﻿using AOT;
+using System;
+using System.Collections.Generic;
+using System.Runtime.InteropServices;
+using Microsoft.MixedReality.Toolkit.Extensions.Webrtc.Delegates;
+
+namespace Microsoft.MixedReality.Toolkit.Extensions.Webrtc.Marshalling
+{
+    /// <summary>
+    /// A peer, backed by a native-library implementation
+    /// </summary>
+    public class NativePeer : IPeer
+    {
+        /// <summary>
+        /// Internal marshalling definitions to the native-library
+        /// </summary>
+        private static class NativeMethods
+        {
+#if UNITY_ANDROID
+            private const string dllPath = "libjingle_peerconnection_so";
```

Comment:
>libjingle_peerconnection_so [](start = 44, length = 27)

is this the standard webrtc binary for android? Or is this built out of some other repo?